### PR TITLE
Add example for LibreNMS

### PR DESCRIPTION
**Pull request by jacobw**

### CODE REVIEW

File: example_configs/librenms.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+## Essential
+
+## auth_ldap_uid_attribute
+
+```    
+uid
+```
+default: uidNumber
+
+This sets 'uid' as the unique ldap attribue for users.
```
Comment (nitnelave) [inline example_configs/librenms.md:14]: Nit: attribute

File: example_configs/librenms.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+You can either configure LibreNMS from the webui or from the command line. This is a list of the variables that you should set.
+
+## Essential
+
+## auth_ldap_uid_attribute
+
+```    
+uid
+```
+default: uidNumber
```
Comment (nitnelave) [inline example_configs/librenms.md:12]: Including the default is a bit confusing, it looks like it's a recommendation from LLDAP. I would just remove them to clear up any ambiguity

**Review by nitnelave — APPROVED** [review]
Let's just remove the default values, otherwise it looks great, thanks!

File: example_configs/librenms.md (hunk 1 above)
Comment (jacobw) [reply example_configs/librenms.md]: Done.

File: example_configs/librenms.md (hunk 2 above)
Comment (jacobw) [reply example_configs/librenms.md]: Done.

**Review by nitnelave — APPROVED** [review]

**Comment by nitnelave** [comment]
Thank you everyone for the contribution!
This feature is now available in the latest release, [v0.5.0](https://github.com/lldap/lldap/releases/tag/v0.5.0).
You can support LLDAP by starring our repo, contributing some configuration examples and becoming a sponsor.

**Pull request merged**